src/main/java/com/mongodb/jdbc/MongoSQLResultSet.java

```diff
@@ -373,9 +385,15 @@ protected String getString(BsonValue o) throws SQLException {
         if (checkNull(o)) {
             return null;
         }
+        JsonWriterSettings settings = JsonWriterSettings.builder().outputMode(JsonMode.EXTENDED).build();
```

Comment:
The settings and registry can be final static variables, there should be no need to create new ones each time toString is called.

Reply:
Made the change to make them static final variables.